components/ILIAS/Mail/classes/Mime/Sender/class.ilMailMimeSenderUser.php

```diff
@@ -79,19 +79,13 @@ public function getFromName(): string
             return $this->user->getFullname();
         }
 
-        $placeholders = [
-            'FULLNAME' => $this->user->getFullname(),
-            'FIRSTNAME' => $this->user->getFirstname(),
-            'LASTNAME' => $this->user->getLastname(),
-        ];
-
-        $template = $from;
-        $interpolated = $this->mustache_factory->getBasicEngine()->render($template, $placeholders);
-
-        if ($template !== $interpolated) {
-            return $interpolated;
+        $name = str_ireplace('[FULLNAME]', $this->user->getFullname(), $from);
```

Comment:
This looks wrong. With ILIAS 10.x, we need the "Mustache" handling for placeholders in the "Mail Signature".